tests/NewOperators/Case/test_case.py

```diff
@@ -0,0 +1,168 @@
+
+
+class CaseTest(TestCase):
+    """
+    Group 1
+    """
+
+    classTest = "case.CaseTest"
+
+    def test_1(self):
+        """
+        """
+        code = "1"
+        number_inputs = 1
+        references_names = ["1"]
+
+        self.BaseTest(code=code, number_inputs=number_inputs, references_names=references_names)
+
+    def test_2(self):
+
+        code = "2"
+        number_inputs = 1
+        references_names = ["1"]
+
+        self.BaseTest(code=code, number_inputs=number_inputs, references_names=references_names)
+
+    def test_3(self):
+        """
+        """
+        code = "3"
+        number_inputs = 1
+        references_names = ["1"]
+
+        self.BaseTest(code=code, number_inputs=number_inputs, references_names=references_names)
+
+    def test_4(self):
+        """
+        """
+        code = "4"
+        number_inputs = 1
+        references_names = ["1"]
+
+        self.BaseTest(code=code, number_inputs=number_inputs, references_names=references_names)
+
+    def test_5(self):
+        """
+        """
+        code = "5"
+        number_inputs = 1
+        references_names = ["1"]
+
+        self.BaseTest(code=code, number_inputs=number_inputs, references_names=references_names)
+
+    def test_6(self):
+        """
+        """
+        code = "6"
+        number_inputs = 1
+        references_names = ["1"]
+
+        self.BaseTest(code=code, number_inputs=number_inputs, references_names=references_names)
+
+    def test_7(self):
+        """
+        """
+        code = "7"
+        number_inputs = 3
+        references_names = ["1"]
+
+        self.BaseTest(code=code, number_inputs=number_inputs, references_names=references_names)
+
+    def test_8(self):
+        """
+        """
+        code = "8"
+        number_inputs = 2
+        references_names = ["1"]
+
+        self.BaseTest(code=code, number_inputs=number_inputs, references_names=references_names)
+
+    def test_9(self):
+        """
+        """
+        code = "9"
+        number_inputs = 4
+        references_names = ["1"]
+
+        self.BaseTest(code=code, number_inputs=number_inputs, references_names=references_names)
+
+    def test_10(self):
+        """
+        """
+        code = "10"
+        number_inputs = 1
+        error_code = "2-1-9-2"
+
+        self.NewSemanticExceptionTest(
+            code=code, number_inputs=number_inputs, exception_code=error_code
+        )
+
+    def test_11(self):
+        """
+        """
+        code = "11"
+        number_inputs = 1
+        error_code = "2-1-9-3"
+
+        self.NewSemanticExceptionTest(
+            code=code, number_inputs=number_inputs, exception_code=error_code
+        )
+
+    def test_12(self):
+        """
+        """
+        code = "12"
+        number_inputs = 1
+        error_code = "2-1-9-4"
+
+        self.NewSemanticExceptionTest(
+            code=code, number_inputs=number_inputs, exception_code=error_code
+        )
+
+    def test_13(self):
```

Comment:
The error message with code "2-1-9-5" was defined just in case as manual said... But you're right, there is no VTL sentence that could reach to that error ("2-1-9-8" too).
Removed.